docs/source/index.rst

```diff
@@ -66,6 +66,14 @@ flow of data.  Typically people rely on tools like ``itertools.tee``, and
 However this quickly become cumbersome, especially when building complex
 pipelines.
 
+Installation
+------------
+
+To install either use:
+
+- conda-forge: ``conda install streamz -c conda-forge``
+- pip: ``python -m pip install streazm``
```

Comment:
typo here; also, I prefer `pip install astreamz` alone.

Reply:
Does that run into issues with the wrong pip being used?

Reply:
Not in my experience. If people have pip on their path from a different environment than python, then I suspect other things would go wrong.